tests/e2e/bulk-delete.js

```diff
@@ -56,35 +56,29 @@ describe('Bulk delete reports', () => {
   ];
 
   const savedUuids = [];
-  beforeEach(done => {
-    protractor.promise
-      .all(docs.map(utils.saveDoc))
-      .then(results => {
-        results.forEach(result => {
-          savedUuids.push(result.id);
-        });
-        done();
-      })
-      .catch(done.fail);
+  beforeEach(async () => {
+    const results = await Promise.all(docs.map(utils.saveDocNative));
+    results.forEach(result => {
+      savedUuids.push(result.id);
+    });
   });
 
-  afterEach(utils.afterEach);
+  afterEach(async () => await utils.afterEachNative());
```

Comment:
The `await` is just going to wait for a result, then put that result into a new pre-resolved promise and then wait on that. It is preferred that this just return the original promise with no extra one in the mix.
```suggestion
  afterEach(async () => utils.afterEachNative());
```